### PR TITLE
feat: add use cases menu

### LINKED ISSUE #207: Feature: Add `Use Cases` dropdown in navigation bar
### Type of feature

🍕 Feature

### Current behavior

N/A

### Suggested solution

Implement a dropdown menu that leads to various marketing pages
- [ ] For teams: `/teams`
- [ ] Startups: `/startups`
- [ ] Students: `/students`
- [ ] Contributors: `/contributors`
- [ ] Maintainers: `/maintainers`

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by zeucapua**
## Description

Adds a drop down menu that leads to various use cases pages

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Closes #207 

## Mobile & Desktop Screenshots/Recordings

Screenshot 2024-03-12 at 8 15 55 AM

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by jpmcb — COMMENTED** [review]
Screenshot 2024-03-12 at 9 21 50 AM

Is there a `/contributors` page yet? Going there 404s for me in the deploy preview

**Review by nickytonline — COMMENTED** [review]
The nav for use cases works on dekstop, but the contributors page is giving a 404 when I go to it from the use cases menu.

CleanShot 2024-03-12 at 11 21 42

**Comment by nickytonline** [comment]
> Screenshot 2024-03-12 at 9 21 50 AM
> 
> Is there a `/contributors` page yet? Going there 404s for me in the deploy preview

The page appears to be there in code.

**Comment by nickytonline** [comment]
The menu is also not above the page contents.

CleanShot 2024-03-12 at 11 24 32

**Comment by zeucapua** [comment]
I think this branch some how has an upstream from earlier before I made the `/contributors` page, but it should work if merged into main since that has the page @nickytonline

File: components/sections/navigation/DesktopNavigation.tsx [hunk 1]
```diff
@@ -35,7 +54,22 @@ const DesktopNavigation: FC<DesktopNavigationProps> = ({
         </Link>
       </div>
 
-      <nav className="flex gap-8">
+      <nav className="flex gap-8 items-center">
+        <Menu as="div" className="relative -top-0.5">
+          <Menu.Button className={`${defaultStyle}`}>Use Cases</Menu.Button>
+          <Menu.Items className="rounded-lg absolute left-0 w-96 flex flex-col gap-4 p-3 shadow-xl origin-top-left bg-darkBG z-50 my-4">
+            { useCases.map((useCase) => (
+              <Menu.Item>
```
Comment (nickytonline) [inline components/sections/navigation/DesktopNavigation.tsx:62]: refactor (blocking): The `key` prop is missing.

**Comment by isabensusan** [comment]
+1 to /contributors page throwing a 404 (maybe the deploy preview hasn't updated yet)
+1 to fixing the z-index for the dropdown to be above anything else

**Mobile**
one more thing is we need to add the links to the mobile navigation -- we don't need to add the full dropdown here, I think just adding the links "Contributors" "Maintainers" "Teams" would be ok for mobile
image

**Review by jpmcb — APPROVED** [review]
👏🏼 👏🏼

File: components/sections/changelog/Hero.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Heading, Typography } from '../../common/text'
 
 const Hero = () => {
   return (
-    <section className="z-50 py-24 tablet:py-56 border-slate-800 relative flex flex-col gap-y-5 from-transparent via-red-800 to-transparent">
+    <section className="z-40 py-24 tablet:py-56 border-slate-800 relative flex flex-col gap-y-5 from-transparent via-red-800 to-transparent">
```
Comment (nickytonline) [inline components/sections/changelog/Hero.tsx:6]: We can revisit the z-index issue, but this is fine for the launch.

**Review by nickytonline — COMMENTED** [review]
All's working great. I think all that's missing is @isabensusan's request for links for mobile.

**Review by nickytonline — APPROVED** [review]
Thanks @zeucapua! 🚢

**Pull request merged**